### ISSUE TITLE
The fontWeight dont work

### ISSUE THREAD

**Issue by MBoukhames** [google/charts#351]
Hi, 
fontWeight from TextStyleSpec is apparently not used.
How to style the text?

**Comment by Carlit0** [comment]
Yes, setting custom **fontWeight** and **fontFamily** in TextStyleSpec are not working.

**Comment by andreddie81** [comment]
is this fixed yet??

**Comment by agilob** [comment]
Ahhh... spent over 30 minutes trying to set font to bold :D

**Comment by InitAlpha** [comment]
Same here, and it's of a String type...weird.

**Comment by benznest** [comment]
+1 same here. FontFamily in TextStyleSpec is not working.

**Comment by benPesso** [comment]
The Flutter Charts library in general is lacking a lot of documentation. I came across at least 5-6 conflicts and undocumented issues while implementing it.

**Comment by agilob** [comment]
@benPesso  it would be nice if you could at least open issue for each of them to keep track.

**Comment by benPesso** [comment]
@agilob Will do. Once I'm done with the implementation, and have a full list. :)

**Comment by andreddie81** [comment]
seriously guys...spend some more time in the documentation....it is terrible!

**Comment by mellowcello77** [comment]
Yip, the graphs can be nice, really hard to find simple things like this on it.

**Comment by Simperfy** [comment]
Is there any update regarding this issue? I'm working on a chart and I need to change the font weight.

**Comment by marcel-bluestone** [comment]
same here....

**Comment by 4rgc** [comment]
This has been here almost for a year...
Hope this project gets revived soon!

**Comment by marimeli** [comment]
February 2021 and I found that fontWeight  is still not working. :(

**Comment by chris** [comment]
Same here, as looking at the code the `fontWeight` value is just never used. It'd also be nice if we could use standard Flutter items for this, so `FontWeight.bold`.

**Comment by huantd0111** [comment]
Hope the development team fixes this problem soon!

**Comment by juanagu** [comment]
news?

**Comment by TMAnhoangthinh** [comment]
Any updates?

**Comment by raphasauer** [comment]
Has anyone found a way to use a bold font in the title?

**Comment by altafkhan8719** [comment]
any update??,  is the issue resolved???

**Comment by darkstarx** [comment]
No updates in the google's repository.
You are welcome to use my fork where this problem is solved.
```
dependencies:
  charts_flutter:
    git:
      url: git@github.com:darkstarx/charts.git
      path: charts_flutter
      ref: master
```
